analytics/views.py

```diff
@@ -761,27 +785,32 @@ def user_activity_summary_table(user_summary):
     return make_table(title, cols, rows)
 
 def realm_user_summary_table(all_records, admin_emails):
+    # type: (List[QuerySet], Set[str]) -> Tuple[Any, str]
```

Comment:
I think the first return value, `user_records`, you can get the type from what `get_user_activity_summary` returns